assets/js/cycles.js

```diff
@@ -23,5 +23,25 @@ var Cycles = {
                 .replace('{month}', Language.get(`menu.month.${_date[0]}`))
                 .replace('{day}', _date[1])
         );
+        return _date[1];
     }
-}
\ No newline at end of file
+}
+
+// show alert when cycles are not up to date
+setInterval(function () {
+    var newDate = new Date(),
+        day = newDate.getUTCDate(),
+        hour = newDate.getUTCHours(),
+        minute = newDate.getUTCMinutes();
+    
+    if (hour < 1 && minute <= 20)
+        $('.map-alert').css('opacity', '1');
+    
+    if (day != Cycles.setLocaleDate()) {
+        $('.map-alert').css('opacity', '1');
+        return;
```

Comment:
Not needed as there's nothing after it to do. Allows you to remove the `{` and `}` as well.

Reply:
i added return and braces because first i wanted to translate position